### ISSUE TITLE
Rename to sparks?

### ISSUE THREAD

**Issue by t-bast** [lightning/blips#3]
Now that the spec is called `lightning/bolts`, I'd find it really cool if this repo was `lightning/sparks`.
It could stand for "SPecifications for Additional Really Kool Stuff/Software".

**Comment by Bosch-0** [comment]
Much better name <3

**Comment by t-bast** [comment]
My line of thinking with sparks is that it's a tiny start of a lightning bolt, which is what we describe here: much smaller things than bolts, that could potentially evolve later into a bolt (it remains to be seen if that "upgrade" path makes sense and is ever used).

I don't like blast, which sounds "bigger" than bolts, which would give the wrong idea about what we're doing here IMHO.

**Comment by TheBlueMatt** [comment]
I feel like the "are bLIPs a pre-BOLT, or something different (ie just a vendor standard)" question is the question here, and I'm not a fan of renaming to imply one or the other yet.

**Comment by t-bast** [comment]
Not necessarily, I quite like spark mostly because it's smaller than bolts, a small addition to the ecosystem (and dislike blast because it's bigger), regardless of whether it may or may not become a bolt afterwards?

**Comment by cryptagoras** [comment]
Hmm, I can be wrong but from what I understand bLIPs are about 2 things:
i) optional BOLT-like features that could potentially graduate to proper BOLTs if they prove their worth, and
ii) application design docs that utilize one or multiple BOLTs/bLIPs.
So they can be both on the same level like BOLTs (but optional), and on a higher/application level as well.

Unless I misunderstood and this is strictly for potential BOLT-candidates which certainly would be a more focused approach, however there's already [bLIP 10](https://github.com/lightning/blips/blob/master/blip-0010.md) which is mostly on the app-side.

Btw, the only real plus of `blasts` is that it contains the letter B to include bitcoin :smile:

**Comment by cryptagoras** [comment]
I see, yeap, I initially got (i) from [Ryan's bLIPs intro/proposal on lightning-dev ML](https://lists.linuxfoundation.org/pipermail/lightning-dev/2021-June/003086.html), although I didn't translate it precisely.
Naming aside, it's pretty certain that (i) apps bLIPs will be the ruling category since the lower bar, so it's not much of an issue.

Anyhow, I don't mind about the name, as much as having a fully capitalized word... that lowercase `b` is OCD-inducing :zipper_mouth_face:

**Comment by jb55** [comment]
yes
